ui/ui-config/pages/session/sign-in.tsx

```diff
@@ -26,71 +26,112 @@ export default function SignInPage(props) {
     setLoading(false);
 
     if (response?.user?.email) {
-      router.push(nextPage ? nextPage.toString() : "/");
+      if (nextPage) {
+        router.push(nextPage.toString());
+      } else {
+        const { setupSteps, toDisplay } = await getSetupSteps();
+        const isSetupComplete = setupSteps.every((step) => step.complete);
+        if (isSetupComplete || !toDisplay) {
+          router.push("/profiles");
+        } else {
+          router.push("/setup");
+        }
+      }
     }
   };
 
+  async function getSetupSteps() {
+    const { setupSteps, toDisplay }: Actions.SetupStepsList = await execApi(
+      "get",
+      `/setupSteps`
+    );
+    return { setupSteps, toDisplay };
+  }
+
   return (
     <>
       <Head>
-        <title>Grouparoo: Sign In</title>
+        <title>Grouparoo</title>
       </Head>
 
-      <h1>Register</h1>
+      <div style={{ width: "100%", height: "100%" }}>
+        <Row>
+          <Col style={{ textAlign: "center" }}>
+            <Image
+              style={{ maxHeight: 50, margin: 40 }}
+              src="/public/images/logo/logo-and-wordmark-black-words.svg"
+            />
+            <br />
+            <h2>Sync, Segment, and Send your Product Data Everywhere</h2>
+            <p>
+              Grouparoo is open source data infrastructure to keep customer data
+              in-sync between your data warehouse and 3rd party tools.
+            </p>
+            <p>
+              Learn more at{" "}
+              <a target="_blank" href="https://www.grouparoo.com">
+                www.grouparoo.com
+              </a>
+            </p>
+            <br />
+          </Col>
+        </Row>
```

Comment:
Linking the thread from https://github.com/grouparoo/grouparoo/pull/1835/files#r648408807 - now that there aren't redirects to sign in from `/`, is this content needed?